data/json/items/armor/arms_armor.json

```diff
@@ -138,12 +138,10 @@
       {
         "covers": [ "arm_l", "arm_r" ],
         "encumbrance": 5,
-        "coverage": 90,
-        "cover_melee": 90,
-        "cover_ranged": 95,
-        "cover_vitals": 90,
```

Comment:
Can't delete vitals coverage here, it defaults to 0 so having it not defined is bad. 

Also for this one the plastic shouldn't be 100% coverage though the neoprene can be. You can do this by specifying the material coverages. I think 90% covered by plastic is reasonable. 

I'm on mobile rn but could provide an example in a few hours.

Reply:
I re-added the vitals coverage as well as specifying varying material coverages for the plastic. Should be good now.